packages/next/pages/_document.tsx

```diff
@@ -371,7 +371,11 @@ export class Main extends Component {
   render() {
     const { amphtml, html } = this.context._documentProps
     if (amphtml) return '__NEXT_AMP_RENDER_TARGET__'
-    return <div id="__next" dangerouslySetInnerHTML={{ __html: html }} />
+    return (
+      <>
+        <div id="__next" dangerouslySetInnerHTML={{ __html: html }} />
```

Comment:
Why have this nested in a fragment when it's just one element?

Reply:
Ops 😛
Fixed it on #7441